packages/sanity/src/core/form/studio/assetSource/DefaultSource.tsx

```diff
@@ -58,7 +102,16 @@ const DefaultAssetSource = function DefaultAssetSource(
     dialogHeaderTitle = 'Select image',
     onClose,
     onSelect,
+    accept,
   } = props
+  const acceptTypes = accept
+    ? accept
+        .split(',')
+        .map((a) => a.trim())
+        .join(', ')
+    : ''
+  const showAcceptMessage = !isLoading && accept && accept.length > 0
+  const isImageOnlyWildCard = accept && accept === 'image/*' && assetType === 'image'
```

Comment:
Needed otherwise, the default of the "image input" always returning "image/*" would trigger the accept message to always appear. And in that use case it is not necessary.